docs/upcoming_changes/9346.deprecation.rst

```diff
@@ -0,0 +1,8 @@
+Explicitly setting ``NUMBA_CAPTURED_ERRORS=old_style`` will raise deprecation warnings
+======================================================================================
+
+As per deprecation schedule of old-style error-capturing, explicitly setting 
+``NUMBA_CAPTURED_ERRORS=old_style`` will raise deprecation warnings. 
+This release is the last to use "old_style" as the default.
+Details are documented at 
+https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-old-style-numba-captured-errors
```

Comment:
Should this reference a specific release which has this deprecation notice anchor, else this link from the release notes to the docs will eventually become invalid?

Reply:
i can make it reference the last release

Reply:
resolved in https://github.com/numba/numba/pull/9346/commits/b326582276620e74b48192422be184961363aa3e

Reply:
Thanks.